### ISSUE TITLE
[Accessibility] In settings->about, the content (version number) is not accessible.

### ISSUE THREAD

**Issue by liuche** [mozilla-mobile/firefox-tv#580]
Feedback from Eitan that this page isn't being read by screenreader. Not a blocker for 2.0, so moving this out of the milestone.
